schema.graphql

```diff
@@ -68,14 +68,20 @@ type Project @entity {
   distributeToTicketModEvents: [DistributeToTicketModEvent!]!
     @derivedFrom(field: "project")
   deployedERC20s: [DeployedERC20Event!]! @derivedFrom(field: "project")
-  distributePayoutsEvents: [DistributePayoutsEvent!]! @derivedFrom(field: "project")
-  distributeToPayoutSplitEvents: [DistributeToPayoutSplitEvent!]! @derivedFrom(field: "project")
-  distributeReservedTokensEvents: [DistributeReservedTokensEvent!]! @derivedFrom(field: "project")
-  distributeToReservedTokenSplitEvents: [DistributeToReservedTokenSplitEvent!]! @derivedFrom(field: "project")
+  distributePayoutsEvents: [DistributePayoutsEvent!]!
+    @derivedFrom(field: "project")
+  distributeToPayoutSplitEvents: [DistributeToPayoutSplitEvent!]!
+    @derivedFrom(field: "project")
+  distributeReservedTokensEvents: [DistributeReservedTokensEvent!]!
+    @derivedFrom(field: "project")
+  distributeToReservedTokenSplitEvents: [DistributeToReservedTokenSplitEvent!]!
+    @derivedFrom(field: "project")
   deployedERC20Events: [DeployedERC20Event!]! @derivedFrom(field: "project")
   useAllowanceEvents: [UseAllowanceEvent!]! @derivedFrom(field: "project")
   projectEvents: [ProjectEvent!]! @derivedFrom(field: "project")
   ethErc20ProjectPayers: [ETHERC20ProjectPayer!]! @derivedFrom(field: "project")
+  deployedVeNftEvents: [DeployedVeNftEvent!]! @derivedFrom(field: "project")
+  veNftContract: VeNftContract! @derivedFrom(field: "project")
```

Comment:
will a project only ever have one `veNftContract` ?

Reply:
As far as I know it's 1-to-1 right now from conversations.